cg/apps/janus/api.py

```diff
@@ -0,0 +1,28 @@
+"""The JanusAPIClient."""
+
+from http import HTTPStatus
+
+import requests
+from requests import Response
+
+from cg.apps.janus.dto.create_collect_qc_request import CreateCollectQCRequest
+from cg.exc import JanusAPIError
+
+
+class JanusAPIClient:
+    def __init__(self, config: dict[str]):
+        self.host = config["janus"]["host"]
+
+    def get_qc_metrics(self, collect_qc_request: CreateCollectQCRequest):
+        endpoint: str = f"{self.host}/collect_qc_metrics"
+        response = requests.post(endpoint, data=collect_qc_request)
```

Comment:
The request data needs to be serialized, you need to do
```python
data = collect_qc_request.model_dump_json()
```

Reply:
fixed